packages/block-library/src/post-featured-image/index.php

```diff
@@ -19,7 +19,8 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 	}
 	$post_ID = $block->context['postId'];
 
-	$featured_image = get_the_post_thumbnail( $post_ID );
+	$sizeSlug       = isset( $attributes['sizeSlug'] ) ? $attributes['sizeSlug'] : 'post-thumbnail';
```

Comment:
I decided to fall back to `post-thumbnail`, so this change doesn't break current behavior.